### PR TITLE
Add my SSB mon into the SSB code

### PR DESCRIPTION
**Pull request by DesoGit**

### CODE REVIEW

**Comment by Mystifi** [comment]
@HoeenCoder, does this look right?

File: mods/sgssb/scripts.js [hunk 1]
```diff
@@ -4,21 +4,21 @@
 
 exports.BattleScripts = {
     randomSeasonalRegStaffTeam: function (side) {
-        var team = [];
```
Comment (HoeenCoder) [inline mods/sgssb/scripts.js:7]: Why change from 'var' to 'let'?

File: mods/sgssb/scripts.js [hunk 2]
```diff
@@ -4,21 +4,21 @@
...
+        let team = [];
+        let variant = this.random(2);
 
-        var sets = {
+        let sets = {
             // Admins.
             '~HoeenHero': {
                 species: 'Ludicolo', ability: 'Swift Swim', item: 'Leftovers', gender: 'M',
-                moves: [['Hydro Pump', 'Scald'][this.random(2)], 'Ice Beam', 'Giga Drain'],
+                moves: [['Hydro Pump', 'Scald'][variant], 'Ice Beam', 'Giga Drain'],
```
Comment (HoeenCoder) [inline mods/sgssb/scripts.js:14]: 'variant' is for random genders. It would work here though.

**Review by HoeenCoder — COMMENTED** [review]
Mostly just concerns. Also your custom move was already merged right?

**Comment by HoeenCoder** [comment]
Ok confirmed with @DesoGit that the move is merged, and talked to him about the changes. This will work fine.

**Pull request merged**